src/dream.mli

```diff
@@ -1387,7 +1387,14 @@ val session_label : request -> string
 val session_expires_at : request -> float
 (** Time at which the session will expire. *)
 
+(** {1 Flash Messages} *)
 
+val flash_messages : ?lifetime:float -> middleware
```

Comment:
Is `?litetime` necessary? We might be able to just hard-code it.

Reply:
That sounds fine to me.